chat/src/main/java/com/github/twitch4j/chat/TwitchChat.java

```diff
@@ -108,7 +108,19 @@
     /**
      * Channel Cache
      */
-    protected final Set<String> channelCache = ConcurrentHashMap.newKeySet();
+    protected final Set<String> currentChannels = ConcurrentHashMap.newKeySet();
+
+    /**
+     * Cache: ChannelId to ChannelName
+     */
+    @Getter
+    protected final Map<String, String> channelIdToChannelName = new ConcurrentHashMap();
```

Comment:
Could add `<>` in the assignment portion to avoid the raw usage warning (same applies for the second map)